src/web_app/js/call.js

```diff
@@ -18,6 +18,40 @@
 
 var Call = function(params) {
   this.params_ = params;
+
```

Comment:
I do not want to store them unescaped in the DOM but I will try to escape them earlier.